src/main/js/controls/Timeline/Timeline.js

```diff
@@ -277,6 +283,54 @@ class Timeline extends Construct {
         return this;
     }
 
+    /**
+     * Updates the graph axisData and content.
+     *
+     *  @returns {Timeline} - Timeline instance
+     * @param {Array} graphData - Input array that holds updated values and key
+     */
+    reflow(graphData) {
```

Comment:
1. Looks like we have the same method `reflow` in two different places with different implementations.

-  src/main/js/controls/Timeline/Timeline.js
-  src/main/js/controls/Timeline/TimelineContent.js


2. Why are we having the same method  `reflow`  name in two different places?

Reply:
The reflow function in both the files is targeted at different configs. The reflow in Timeline.js is used to update the construct and the reflow in the TimelineContent.js is used to update the graph content. 

Both have the same name because what they are doing is similar functionality i.e to update the config. Its is similar to why both files have the `resize` function